examples/feature_interface/partcad.yaml

```
@@ -215,7 +215,7 @@ parts:
       m4-slotted-30-thru-3:
         L: # top left
           location: [[-15.0, -27.0, 27.0], [1.0, 0.0, 0.0], 90.0]
-          params: {offset: -15}
+          params: { offset: -15 }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)